test/lenses.js

```diff
@@ -41,16 +56,24 @@ describe('view, over, and set', function() {
 
   they('may be applied to composed lenses', function() {
     var streetLens = R.compose(addressLens, headLens);
+    var dogLens = R.compose(R.lensPath(['pets']), R.lensPath(['dog']));
+
+    eq(R.view(dogLens, alice), R.view(R.lensPath(['pets', 'dog']), alice));
```

Comment:
@CrossEye actually `test/lenses.js` is about `view`, `over`, and `set`. I am very willing to try to do some refactoring and add specific tests for lensIndex, lensProp and lensPath outside of `test/lenses.js` but preferably not as part of this PR.

Would you prefer if I did that first ?